### DISCUSSION TITLE
xtb killed with larger molecules

### DISCUSSION THREAD

**Discussion by Erikna00** [grimme-lab/xtb#811] [Q&A]
Hi,

I have an issue where when i try to run larger molecules (read proteins, 5-40 kDa) xtb gets killed. No error message just "Killed" appearing in the command line. My only lead on this is that the process is always killed when RAM usage goes to about 16 GB. does anyone know what is at fault and how to fix it?

run prompts
export MKL_NUM_THREADS=10
export OMP_NUM_THREADS=10,1
export OMP_STACKSIZE=15G
ulimit -s unlimited
xtb dipent-light.xyz > dipent-light.out  --gfn 1 --scc

Specs
AMD ryzen 5 3600
32 GB DDR4 RAM 3600 MHz
Running xtb in WSL2 ubuntu on windows 10

And before anyone calls me stupid for using xtb on proteins, my problem is simulating covalentlly bound ligands, something i have had great trouble getting GROMACS to do since making the .top and .gro with correct parameters when those are taken from ligpargen is beyond me. Besides, using DFT to see if the covalent bond will dissociate is quite useful.

**Comment by haneug** [comment]
Without an explicit example, we can not reproduce your problem. Maybe the RAM in your Ubuntu on Windows is limited to 16 G? Also, note that GFN1-xTB is not a DFT method, but an extended-tight-binding theory that approximates DFT.

**Reply by ghost** [reply]
Same here !
I have the same problem.
Is the memory
The task is killed by out of memory.
